content/pages/meetup/2023/0816-nycu.rst

```diff
@@ -0,0 +1,80 @@
+========================================
+Meetup 2023 August 16th in NYCU
+========================================
+
+:date: 2023-08-10 20:00
+:url: meetup/2023/0816-nycu
+:save_as: meetup/2023/0816-nycu.html
+
+Date
+-----
+
+* Date: August 16th, 2023
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+* 18:30-19:30 Free discussion
+* 19:30-20:30 Discussion: Review for August scisprint
+* 20:30-21:30 Off-topic discussion
+
+|
+
+Discussion: Review for August scisprint
+----------------------------------------
+
+We will review the August scisprint to enhance our workflow and plan 
+for the upcoming September scisrpint.
+
+* 19:30-20:05 (35min) Review the August scisprint
+* 20:05-20:30 (25min) Preparation for September scisprint
+
+|
+
+Sign up
+------------
+
+The event is free. Please register on kktix.
```

Comment:
I just create the kktix event. The link is :
https://sciwork.kktix.cc/events/meetup-20230816

Reply:
Thanks a lot. I have updated in the page.